source/wp-content/themes/wporg-news-2021/blocks/month-in-wp-title/index.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace WordPressdotorg\Theme\News_2021\Blocks\Month_In_WordPress;
+
+add_action( 'init', __NAMESPACE__ . '\register_block' );
+add_action( 'enqueue_block_assets', __NAMESPACE__ . '\register_block_type_js' );
+
+/**
+ * Renders the `wporg/month-in-wp-title` block on the server.
+ *
+ * @param array    $attributes Block attributes.
+ * @param string   $content    Block default content.
+ * @param WP_Block $block      Block instance.
+ *
+ * @return string Returns the release version for the current post wrapped inside "h1" tags.
+ */
+function render_block( $attributes, $content, $block ) {
+	if ( ! isset( $block->context['postId'] ) ) {
+		return '';
+	}
+
+	$post_id  = $block->context['postId'];
+	$tag_name = 'h2';
+
+	if ( isset( $attributes['level'] ) ) {
+		$tag_name = 0 === $attributes['level'] ? 'p' : 'h' . $attributes['level'];
+	}
+
+	$date = date_create_from_format( 'Y-m-d H:i:s', get_the_date( 'Y-m-00 00:00:00', $post_id ) );
```

Comment:
`'Y-m-00 00:00:00'`

Is this what makes it grab the _previous_  month instead of the month that the post was actually published in? If so, I think it would be worth documenting that in a code comment, as it doesn't stand out that that's what's happening here.

Reply:
Yep, that's what's going on. Updated with a comment in both files this appears https://github.com/WordPress/wporg-news-2021/pull/290/commits/f8d5054833274d0d5e778f9e41a89f852d6542d2